### PR TITLE
Add policy tag for modules

### PR DESCRIPTION
**Pull request by michael-kernel-sanders**

### CODE REVIEW

File: oak/proto/policy.proto [hunk 1]
```diff
@@ -47,3 +48,11 @@ message GrpcTag {
   // https://github.com/google/asylo/blob/3158887cb768112516424b3e65046f3946eb4465/asylo/identity/identity.proto#L40.
   bytes authorization_bearer_token = 1;
 }
+
+// Policies related to modules, referring to the native WebAssembly node within the TCB.
+message ModuleTag {
```
Comment (tiziano88) [inline oak/proto/policy.proto]: I think the name should make it clear that this is related to WebAssembly modules specifically. How about `WebAssemblyModuleTag`?

File: oak/proto/policy.proto [hunk 2]
```diff
@@ -47,3 +48,11 @@ message GrpcTag {
...
+  // The attestation for a single WebAssembly module.
```
Comment (tiziano88) [inline oak/proto/policy.proto]: It is unclear what an "attestation for a module is". Perhaps mention something like it is the sha256 of the module in binary format, or something like that? It's fine if we change our mind later, but currently it seems very abstract.

File: oak/proto/policy.proto (hunk 1 above)
Comment (michael-kernel-sanders) [reply oak/proto/policy.proto]: Done.

File: oak/proto/policy.proto (hunk 2 above)
Comment (michael-kernel-sanders) [reply oak/proto/policy.proto]: Done.

**Pull request merged**